### DISCUSSION TITLE
Crates within Crates

### DISCUSSION THREAD

**Discussion by gavrielsha** [WebCrateApp/webcrate#28] [Ideas]
I love your tool! It would be the best if I could categorize certain crates into broader crates, i.e. creating crates within larger crates

Example: Design> Tools> Prototyping

Reference: https://makechaos.app/features

**Comment by BetaHuhn** [comment]
Interesting, have never thought about this.

I personally don't like nesting stuff and prefer something like [tags](https://github.com/WebCrateApp/webcrate/discussions/24) instead.

Would that be something that would work for you instead?

(Not a rejection, I'm just looking to see what people need the most and what to focus on next)

**Comment by gavrielsha** [comment]
Tags would be awesome actually. Also importing would be great too.

**Reply by BetaHuhn** [reply]
Great, will bump tags up the list and work on it soon!

**Reply by BetaHuhn** [reply]
Also, I'm currently looking to do some user research for WebCrate and I would love to talk to you 1:1 to gather some feedback and float some ideas around.

Would you be interested in that @gavrielsha?

**Comment by gavrielsha** [comment]
Sure, I'd be happy to chat. When are you free?

**Reply by BetaHuhn** [reply]
Awesome! I'll shoot you an email to talk details!

**Comment by eshate** [comment]
Hey! I just wanted to say, that I really like your work - I was looking for something simple and minimalistic like WebCrates, appreciate. 
Also - found that topic, because I want to bump the idea of subcrates/tags. If you're working on it - that's cool. I am also open for sharing some ideas if you still want to talk with people about it! See ya!

**Reply by BetaHuhn** [reply]
Hey, thanks for the nice words!

Tags is something I am still interested in adding in the near future, I just haven't had the time to start working on it yet. It will be a part of one the upcoming releases for sure.

Any type of feedback is always appreciated, feel free to open a discussion with your ideas! I am also happy to talk to you directly, either via email or in a short call if that is something you are okay with! Just shoot me an email at hi@webcrate.app if you are interested 😄
